src/cli.ts

```diff
@@ -66,7 +67,10 @@ process.on('unhandledRejection', handler);
 
 
 (async () => {
-  const electronPrebuiltPath = argv.e ? path.resolve(process.cwd(), argv.e) : locateElectronPrebuilt();
+  const projectRootPath = getProjectRootPath(process.cwd());
+  console.log(`projectRootPath: ${projectRootPath}`);
```

Comment:
Please don't leave these stray logs in

Reply:
@MarshallOfSound  This pr stay for a long time, Can you merge it now?

Reply:
Would love to see this merged as we're currently struggling to use electron-forge in our monorepo due to issues with electron-rebuild. Is there something blocking this merge, anything I could do to help out? @MarshallOfSound